CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
```

Comment:
Can we update the device reference docs to explain how we're distinguishing devices this way?